Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiRequestBus.h

```diff
@@ -104,6 +104,10 @@ namespace AzToolsFramework::ViewportUi
         virtual void SetTextFieldVisible(TextFieldId textFieldId, bool visible) = 0;
         //! Create the highlight border with optional back button to exit the given editor mode.
         virtual void CreateViewportBorder(const AZStd::string& borderTitle, AZStd::optional<ViewportUiBackButtonCallback> backButtonCallback) = 0;
+        //! Check if the viewport border is visible.
+        virtual bool const ViewportBorderVisible() = 0;
+        //! Changes the text on the viewport border while in component mode.
+        virtual void ChangeViewportBorderText(AZStd::string borderTitle) = 0;
         //! Remove the highlight border.
```

Comment:
nit: This would be `Removes viewport border` if we can change it while we're here 🙂